implementations/rust/ockam/ockam/src/workers/echoer.rs

```diff
@@ -0,0 +1,19 @@
+use crate::{Context, Result, Routed, Worker};
+use ockam_core::compat::boxed::Box;
+use ockam_core::compat::string::String;
+use ockam_core::println;
+
+pub struct Echoer;
+
+#[crate::worker]
+impl Worker for Echoer {
+    type Context = Context;
+    type Message = String;
+
+    async fn handle_message(&mut self, ctx: &mut Context, msg: Routed<String>) -> Result<()> {
+        println!("Address: {}, Received: {}", ctx.address(), msg);
```

Comment:
I'm not 100% sure this is appropriate to be in the `ockam` crate -- It's generally bad for library code to println without indication. Can you elaborate on the motivation here?

Reply:
I'll replace it with log message